src/main/drivers/accgyro_mpu6500.h

```diff
@@ -24,9 +24,14 @@
 #define ICM20602_WHO_AM_I_CONST             (0x12)
 
 #define MPU6500_BIT_RESET                   (0x80)
-#define MPU6500_BIT_INT_ANYRD_2CLEAR        (1 << 4)
-#define MPU6500_BIT_BYPASS_EN               (1 << 0)
-#define MPU6500_BIT_I2C_IF_DIS              (1 << 4)
+
+// Register 0x37/55 - INT_PIN_CFG / Pin Bypass Enable Configuration
+#define MPU6500_BIT_RESERVED                1 << 0
```

Comment:
@hydra , @mikeller , these lines are in error. All constants that are `#define`d that include operators **MUST** be surrounded by brackets. This is because they may be used in combination with other operators of higher precedence. So, for example, here `~ MPU6500_BIT_BYPASS_EN` will be interpreted as `~1 << 4`.

This can result in very hard to track down bugs.

@mikeller can you fix this for Betaflight.